spot_driver/test/include/spot_driver/mock/mock_tf_interface.hpp

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
 
+#pragma once
+
 #include <gmock/gmock.h>
 
+#include <geometry_msgs/msg/transform_stamped.hpp>
 #include <spot_driver/interfaces/tf_interface_base.hpp>
+#include <tl_expected/expected.hpp>
```

Comment:
ok, my suggestion to add `tl_expected` to this file must have preceded that being removed from this file? or I just made a mistake because it's not used in this header anymore

Reply:
Yeah after #275 it's not needed, this file changed when I merged `main`.